tests/ui/sanitize/sanitizer-dataflow-basic.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(non_camel_case_types)]
+// Verifies that labels are propagated through loads and stores.
+//
+//@ needs-sanitizer-dataflow
+//@ run-pass
+//@ compile-flags: -Zsanitizer=dataflow -Zsanitizer-dataflow-abilist=src/llvm-project/compiler-rt/lib/dfsan/libc_ubuntu1404_abilist.txt,src/llvm-project/compiler-rt/lib/dfsan/done_abilist.txt,tests/ui/sanitize/sanitizer-dataflow-abilist-rust-no-core-no-std.txt
```

Comment:
Can `-Zsanitizer=dataflow` be made to imply some default abilists?

So you would use `-Zsanitizer-dataflow-abilist=...` if you wanted something other than the default abilist files.

Reply:
I think for this we'll need to wait until the work for default ignorelists is done for all sanitizers and support for default ignorelists is added. What do you think?

Reply:
I think the default ignorelist is front-end specific. Clang already provides default abilists.

Or are you saying this can wait until someone does default lists for all rust sanitizers?

Reply:
I've to add support for having default ignorelists to the frontend and add the default ignorelists for all sanitizers. I'm planning to do it later as part of our work in organizing support for sanitizers (see our [roadmap](https://hackmd.io/@rcvalle/S1Ou9K6H6)). I don't think this PR should block on this; otherwise, it'll take some time for it to be merged.